src/main/java/emissary/client/response/Directory.java

```diff
@@ -0,0 +1,64 @@
+package emissary.client.response;
+
+import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.io.Serializable;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+
+@XmlAccessorType(XmlAccessType.NONE)
+public class Directory implements Comparable<Directory>, Serializable {
```

Comment:
We should see if we can collapse this object with [DirectoryEntryInfo](https://github.com/NationalSecurityAgency/emissary/blob/master/src/main/java/emissary/server/mvc/DumpDirectoryAction.java#L140)...maybe a follow on pr??